app/models/slot.rb

```diff
@@ -1,4 +1,6 @@
-Slot = Struct.new(:date, :start, :end) do
```

Comment:
Is this a rubocop bump / suggestion thing? There are a bunch of these here - if so it would have been super-helpful if they were called out in separate commits.